scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -80,8 +100,9 @@
                  :hostname node-name
                  :resources [{:name "mem" :type :value-scalar :scalar (max 0.0 (:mem available))}
                              {:name "cpus" :type :value-scalar :scalar (max 0.0 (:cpus available))}
-                             {:name "disk" :type :value-scalar :scalar 0.0}]
-                 :attributes []
+                             {:name "disk" :type :value-scalar :scalar 0.0}
+                             {:name "gpus" :type :value-available-types :available-types (:gpus available)}]
```

Comment:
Thoughts on `:value-scalar-map` instead of `:value-available-types`?

Reply:
I agree that `:value-available-types` is not ideal. Not sure about `:value-scalar-map` because that seems slightly confusing. What about just `:value-map`?

Reply:
Discussed offline, landed on `:value-text->scalar`